CHANGELOG.md

```diff
@@ -0,0 +1,130 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [1.0.0] - 28.04.2025 - Author: Mikita Shapel
+
+### Added
+
+- Spring Boot project backbone with pom.xml
+- Setup database schema using Liquibase and configure H2 in-memory database
+- CRUD for Students and Courses
+- Error handler
+- Spring Boot Actuator (health, info)
+- Swagger for REST API
+- Validation for DTOs
+- Job that is triggered daily to collect list of Courses that start tomorrow
+- Logic for sending notification message via email to enrolled Students
+- Logic for coin-based payment
+- Unit and integration testing
+
+### Removed 
+
+- OSIV
+
+## [2.0.0] - 06.05.2025 - Author: Mikita Shapel
```

Comment:
Version must represent actual version of application from pom.xml file (e.g. 1.0.0, 1.1.0 etc.)